lib/bundles/inspec-compliance/api/login.rb

```diff
@@ -0,0 +1,145 @@
+# encoding: utf-8
+# author: Christoph Hartmann
+# author: Dominik Ricter
+# author: Jerry Aldrich
+
+module Compliance
+  class API
+    module Login
+      def login(options)
+        options['server_type'] = Compliance::API.determine_server_type(options['server'], options['insecure']).to_s
+
+        return Login::ComplianceServer.login(options) if options['server_type'] == 'compliance'
+        Login::AutomateServer.login(options)
+      end
+
+      module AutomateServer
+        def self.login(options)
+          verify_thor_options(options)
+
+          options['url'] = options['server'] + '/compliance'
+          token = options['dctoken'] || options['token']
+          puts store_access_token(options, token)
```

Comment:
I realize I'm being a bit pedantic here, but it feels strange to me to have a method called `store_access_token` return a string intended to be user-facing messaging.

I'd rather see `store_access_token` return the `Compliance::Configuration` object it generated to successfully write the config to disk, and then a method that takes that config object and formats a message to the user. It will keep the concerns of the methods succinct and clear. This honestly just reads like a debug message to me. :)

```ruby
config = store_access_token(options, token)

# print out a success message to the user
puts format_status_message(config)
```